creator/events/migrations/0018_add_organizations.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 2.2.20 on 2021-05-13 13:51
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def set_default_org(apps, schema_editor):
+    """
+    Put all existing events under the default organizaton
+    """
+    Organization = apps.get_model("organizations", "Organization")
+    Event = apps.get_model("events", "Event")
+    org = Organization.objects.filter(name="Default Organization").first()
+
+    for event in Event.objects.all().iterator():
+        event.organization_id = org.pk
+        event.save()
+
+
+def delete_default_org(apps, schema_editor):
+    pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("organizations", "0001_initial"),
+        ("events", "0017_add_validation_run_events"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="event",
+            name="organization",
+            field=models.ForeignKey(
+                null=True,
+                help_text="Organization related to this event",
+                on_delete=django.db.models.deletion.CASCADE,
+                to="organizations.Organization",
+            ),
+            preserve_default=False,
+        ),
+        migrations.RunPython(set_default_org, delete_default_org),
```

Comment:
Technically you do not but omitting it will cause it to be an irreversible migration. IMO it's better to still allow the migration to happen and the org and its references to be dropped and thus return the pre-organization setup just as it is now.

Reply:
Oh interesting. I would have thought that since the `reverse_code` is a kwarg, passing in None would just be a no-op. Wonder if by "irreversible" they mean the reverse migration would actually fail with some exception or if they're just advising the developer

Reply:
Trying to downgrade to a migration before this one occurred would raise a `IrreversibleError`. [Docs](https://docs.djangoproject.com/en/3.2/topics/migrations/#reversing-migrations)